src/components/segmentedControl/segment.tsx

```diff
@@ -57,7 +57,7 @@ export type SegmentProps = SegmentedControlItemProps & {
  */
 const Segment = React.memo((props: SegmentProps) => {
   const {
-    activeColor = Colors.primary,
+    activeColor = Colors.$textPrimary,
```

Comment:
`activeColor` is used in both text and icon, currently they are the same but I think this should be reflected in the code.
After looking in `SegmentedControl`, why is there a default here as well?